packages/modules/order/src/migrations/Migration20240604100512.ts

```diff
@@ -32,17 +32,17 @@ export class Migration20240604100512 extends Migration {
     CREATE INDEX IF NOT EXISTS "IDX_order_transaction_return_id" ON "order_transaction" (
```

Comment:
question: nice catch, should the changes to indexes come in a separate migration?

Reply:
These are fine to live here because it will only impact performance and not the functionality.
And I believe before releasing we should have just 1 single migration file for each module.